src/docs/patterns/navigation.mdx

```diff
@@ -131,6 +133,13 @@ A persistent drawer is an always-visible drawer that can be opened and closed (e
         }
         regularWidth
     />
+    <ImageGrid
+        images={[PersistentDrawerPitfallImg, CombinedDrawerImage]}
+        caption={[
+            'Be careful when using persistent drawer with drawer item group headers (where it says "GROUP 1" and "GROUP 2" in the image), as they turn into blank spaces when the drawer collapses.',
+            `One way to get around this issue is to use different drawer variants at different view ports. For example, you can use a permanent drawer on desktop screens (#1) and switch to a persistent drawer without header text or a navigation rail on tablets (#2).`,
+        ]}
```

Comment:
```suggestion
            'When using a persistent drawer with NavGroup titles (where it says "GROUP 1" and "GROUP 2" in the image), the titles become blank spaces when the drawer collapses by default.',
            `If you prefer a look without these spaces, you can conditionally hide the title when the drawer collapses or use different drawer variants at different view ports. For example, you can use a permanent drawer on desktop screens (#1) and switch to a persistent drawer without header text or a navigation rail on tablets (#2).`,
        ]}
```

Reply:
If it's an "issue" (versus a stylistic choice), we should fix it rather than describing a workaround

Reply:
It is more of a design issue rather than a technical one...

Reply:
I would avoid calling it an 'issue' then and say something along the lines of what I suggest above